scripts/cms-any/ci.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+$find = <<<EOT
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    name: CI
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+EOT;
+
+$replace = <<<EOT
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    name: CI
+    # Do not run if this is a pull-request from same repo i.e. not a fork repo
+    # Only check this condition if the account starts with 'silverstripe'
+    if: github.event_name != 'pull_request' || !startsWith(github.event.pull_request.head.repo.owner.login, 'silverstripe') || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
```suggestion
    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
```
Checking if it starts with `silverstripe` isn't necessary, since we will only be updating `ci.yml` on supported modules anyway. Unless your intention is to allow e.g. tractorcow to make PRs to fluent with branches on fluent, and you think there's a reason to have both CI run?

I think there's no reason to let both run IMO. That's the whole point of this work.

Reply:
yeah ... i guess that does make sense, pull-requests on supported modules from forks to forks is a very narrow use case to cater for